docs/basic-features/built-in-css-support.md

```diff
@@ -158,13 +158,22 @@ export default HelloWorld
 
 Please see the [styled-jsx documentation](https://github.com/zeit/styled-jsx) for more examples.
 
-## Sass, Less and Stylus Support
+## Sass Support
 
-To support importing `.scss`, `.sass`, `.less`, or `.styl` files you can use the following plugins:
+Next.js allows you to import Sass using both the `.scss` and `.sass` extensions.
+You can use component-level Sass via CSS Modules and the `.module.scss` or `.module.sass` extension.
```

Comment:
It's compatible with `next-sass` plugin or will `next-sass` plugin be deprecated after these changes? In our case, with `next-sass` we are using `.scss` extensions in component-level and the migration to CSS Modules will be really hard (more than 158 `scss` files + their components).

I guess that will be useful to clarify it in the docs. And maybe providing some painless migration guide... 🙂

Reply:
`next-sass` will be deprecated as soon as these docs land. Updating this should be pretty trivial with a codemod.

Reply:
I didn't know it about codemod before! So this repo https://github.com/zeit/next-codemod is going to be updated with the `next-sass` to `module.sass` transformer?